pkg/apis/app/v2/kieapp_types.go

```diff
@@ -129,6 +129,7 @@ type KieServerSet struct {
 	Database     *DatabaseObject  `json:"database,omitempty"`
 	Jms          *KieAppJmsObject `json:"jms,omitempty"`
 	Jvm          *JvmObject       `json:"jvm,omitempty"`
+	JbpmCluster  *bool            `json:"jbpmCluster,omitempty"`
```

Comment:
Can you add a description of the `JbpmCluster` field above it? Then regenerate the crd and push changes.

Reply:
Done